src/applications/static-pages/facilities/FacilityDetailWidget.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { apiRequest } from '../../../platform/utilities/api';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import { buildAddressArray } from '../../facility-locator/utils/facilityAddress';
+
+export default class FacilityDetailWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    const facilityId = this.props.facilities;
+    this.request = apiRequest(
+      `/facilities/va?ids=${facilityId}`,
```

Comment:
you can probably make your url pass an id as a named param instead of a query param: `/facilities/va/${facilityId}`. this will make it so you get a single result back from the api and you can get rid of the `[0]` from the facility.data[0]` below.